src/extensibility/node/package-validator.js

```diff
@@ -173,6 +173,15 @@ function findCommonPrefix(extractDir, callback) {
 }
 
 /**
+ * Strip the BOM from the file if it has one.
+ * @param {String} data The raw file contents.
+ * @return {String} The same data, sans the BOM.
+ */
+function stripBOM(data) {
+    return data.substr(data.indexOf("{")).replace(/\u0000/g, "");
```

Comment:
We could probably have it as an npm dependency, nothing wrong with that. But also pulling it out into some utils would be fine with me too.